qiita_plugins/target_gene/tgp/tests/test_pick_otus.py

```diff
@@ -66,6 +67,14 @@ def test_generate_pick_closed_reference_otus_cmd(self):
         self.assertEqual(obs, exp)
         self.assertEqual(obs_dir, join(output_dir, 'cr_otus'))
 
+    def test_generate_sortmerna_tgz(self):
+        outdir = mkdtemp()
+        self._clean_up_files.append(outdir)
+        obs = generate_sortmerna_tgz(outdir)
+        exp = ("tar zcf %s/sortmerna_picked_otus.tgz "
```

Comment:
We should never hit the situation where there's spaces in the filepaths right? or if we could maybe worth enclosing the paths between quotes?